### PR TITLE
fix: mobile bottom navigation links

### PR DESCRIPTION
**Pull request by InnaBoitsun**
Fix mobile bottom navigation links

## Related Issue
[AB#62764](https://dev.azure.com/orckestra001/da965986-c85b-43be-9915-219568b000e5/_workitems/edit/62764)
[AB#62798](https://dev.azure.com/orckestra001/da965986-c85b-43be-9915-219568b000e5/_workitems/edit/62798)
[AB#62229](https://dev.azure.com/orckestra001/da965986-c85b-43be-9915-219568b000e5/_workitems/edit/62229)

### CODE REVIEW

File: packages/theme/pages/Category.vue [hunk 1]
```diff
@@ -137,9 +137,9 @@
               :qty="1"
               :is-in-wishlist="isInWishlist({ product })"
               :link="localePath(productGetters.getLink(product))"
-              @input="productsQuantity[product._id] = $event"
+              @input="productsQuantity[product.id] = $event"
               @click:wishlist="!isInWishlist({ product }) ? addItemToWishlist({ product }) : removeProductFromWishlist(product)"
-              @click:add-to-cart="addToCart({ product, quantity: Number(productsQuantity[product._id]) })"
+              @click:add-to-cart="addToCart({ product, quantity: Number(productsQuantity[product.id] ?  productsQuantity[product.id] : 1) })"
```
Comment (oleksandr-honcharuk) [inline packages/theme/pages/Category.vue:142]: productsQuantity[product.id] || 1

File: packages/theme/components/BottomNavigation.vue [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+    const handleAccountClick = async () => {
+      if (isAuthenticated.value) {
+        const localeAccountPath = root.localePath({ name: 'my-account' });
+        return router.push(localeAccountPath);
+      }
+      toggleLoginModal();
+    };
+
+    const handleHomeClick = () => {
+      isMobileMenuOpen.value ? toggleMobileMenu() : false;
```
Comment (oleksandr-honcharuk) [inline packages/theme/components/BottomNavigation.vue:62]: isMobileMenuOpen.value && toggleMobileMenu()

File: packages/theme/components/BottomNavigation.vue [hunk 3]
```diff
@@ -0,0 +1,97 @@
+<template>
+  <!-- TODO: create logic with isActive prop for BottomNavigationItems -->
+  <SfBottomNavigation class="navigation-bottom smartphone-only">
+    <SfBottomNavigationItem
+      :class="route.path == '/' ? 'sf-bottom-navigation__item--active' : ''"
```
Comment (oleksandr-honcharuk) [inline packages/theme/components/BottomNavigation.vue:5]: :class="{ 'sf-bottom-navigation__item--active': route.path == '/'  }"

File: packages/theme/components/BottomNavigation.vue (hunk 2 above)
Comment (InnaBoitsun) [reply packages/theme/components/BottomNavigation.vue:62]: this code comes from defualt theme, I will not change for now.

File: packages/theme/components/BottomNavigation.vue (hunk 3 above)
Comment (InnaBoitsun) [reply packages/theme/components/BottomNavigation.vue:5]: this code comes from defualt theme, I will not change for now.

**Pull request merged**